frontend/awx/interfaces/Schedule.ts

```diff
@@ -32,4 +32,7 @@ export interface Schedule
     modified_by: SummaryFieldsByUser;
     inventory?: SummaryFieldInventory;
   };
+  job_tags: string;
+  skip_tags: string;
+  extra_data: object;
```

Comment:
These items will not live at the top level of a schedule.  The only relation they have to a schedule is through the assigned resource.  Therefore we shouldn't need these interface properties.
```suggestion
```